src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -61,9 +61,9 @@ public enum ErrorCode {
 
   SHRED_BAD_EJSON_VALUE("Bad EJSON value"),
 
-  SHRED_BAD_VECTOR_SIZE("$vector field can't be empty"),
+  SHRED_BAD_VECTOR_SIZE("$vector value can't be empty"),
```

Comment:
Since this is more general reference (used from multiple places), cannot refer as "property", but that is implied by name `$vector` -- hence "value".

Reply:
make sence